sql/core/src/main/scala/org/apache/spark/sql/execution/QueryExecution.scala

```diff
@@ -74,12 +74,23 @@ class QueryExecution(
     sparkSession.sessionState.analyzer.executeAndCheck(logical, tracker)
   }
 
+  lazy val commandCollected: LogicalPlan = analyzed mapChildren { child =>
```

Comment:
`Dataset.logicalPlan` should just call this lazy val?

Reply:
OK